src/telemetry/errno.rs

```diff
@@ -169,3 +169,21 @@ mk_err_enum! {
         _   => UNKNOWN_ERRNO
     }
 }
+
+#[cfg(target_os="windows")]
```

Comment:
This will conflict with my currently open branch [eliza/fix-win](https://github.com/linkerd/linkerd2-proxy/tree/eliza/fix-win), which fixes issue linkerd/linkerd2#1396. I'd like to merge that change first and rebase this onto it, as the code here currently won't compile.

My branch is currently waiting on the reporter to verify the fix, but I can go ahead and open a PR for it so we can get that merged first? I'm pretty sure it does fix the issue, which is a pretty simple syntax error.

Reply:
Opened #54 to get the Windows fix in.